### PR TITLE
Add udev generation and salt script option

### PR DESCRIPTION
**Pull request by ryg-**
bin/make_cfgs.py: added generation of  udev rules
README.md: updated  documentation
network_manager, hw_node.py: added support of salt files in other directories
some updates in tests and test configurations

### CODE REVIEW

File: bin/make_cfgs.py [hunk 1]
```diff
@@ -34,6 +34,11 @@
     INSERT INTO nodes (name, machine_type, mac_address, is_vm, locked, arch, description)
         VALUES  ( 'ses-client-3.a.b.de' , 'client' ,
             'aa:bb:cc:dd:67:16' , 'f' , 'f' , 'x86-64' , '' ) ;
+
+also generated udev rules file for predefined network names setup
+    #ses-client-1
+    SUBSYSTEM=="net", ACTION=="add", DRIVERS=="?*", ATTR{address}=="aa:bb:cc:dd:67.16", ATTR{dev_id}=="0x0", ATTR{type}=="1", KERNEL=="eth*", NAME="eth0"
```
Comment (toabctl) [inline bin/make_cfgs.py]: Is the hardcoded mac there by intention?

File: test_pelagos_teuthology/test_pelagos.py [hunk 2]
```diff
@@ -26,11 +25,10 @@ class TestPelagos(object):
     def setup(self):
         config.load(deepcopy(test_config))
 
-    #    def teardown(self):
+    def teardown(self):
```
Comment (toabctl) [inline test_pelagos_teuthology/test_pelagos.py]: why that if it doesn't do anything?

File: bin/make_cfgs.py (hunk 1 above)
Comment (ryg-) [reply bin/make_cfgs.py]: it is just an example, no more, mac is not real

File: test_pelagos_teuthology/test_pelagos.py (hunk 2 above)
Comment (ryg-) [reply test_pelagos_teuthology/test_pelagos.py]: removed

**Comment by ryg-** [comment]
Commit split as requested

**Pull request merged**